<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertInfo</class>
 <widget class="QWidget" name="insertInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QWidget" name="iw" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>660</height>
    </rect>
   </property>
   <widget class="LabeledEdit" name="w22" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="InteractiveButtonBase" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>570</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="LabeledEdit" name="w42" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="LabeledEdit" name="w31" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="number">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>馆藏数量</string>
    </property>
   </widget>
   <widget class="LabeledEdit" name="w11" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="LabeledEdit" name="w12" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="InteractiveButtonBase" name="summitButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>570</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="LabeledEdit" name="w21" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="LabeledEdit" name="w32" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pdate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>出版日期</string>
    </property>
   </widget>
   <widget class="LabeledEdit" name="w41" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="price">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>价格</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="sendEmailWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>660</height>
    </rect>
   </property>
   <widget class="LabeledEdit" name="emailCodeInput" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="LabeledEdit" name="captchaLabelInput" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="InteractiveButtonBase" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>330</y>
      <width>190</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="InteractiveButtonBase" name="sendButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>发送验证码</string>
    </property>
   </widget>
   <widget class="InteractiveButtonBase" name="cancelButton_2">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>330</y>
      <width>190</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>返回上一页</string>
    </property>
   </widget>
   <widget class="CaptchaBox" name="captchaLabel" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>441</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>邮件验证码将会发送到你的注册邮箱，请注意查收</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveButtonBase</class>
   <extends>QPushButton</extends>
   <header location="global">interactive_buttons/interactivebuttonbase.h</header>
  </customwidget>
  <customwidget>
   <class>LabeledEdit</class>
   <extends>QWidget</extends>
   <header location="global">labeled_edit/labelededit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptchaBox</class>
   <extends>QWidget</extends>
   <header location="global">captchabox/captchabox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>summitButton</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>confirmButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>cancelButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
